scripts/aem.js

```diff
@@ -124,17 +112,8 @@ function setup() {
 
 function init() {
   setup();
-  sampleRUM('top');
-
-  window.addEventListener('load', () => sampleRUM('load'));
-
-  window.addEventListener('unhandledrejection', (event) => {
-    sampleRUM('error', { source: event.reason.sourceURL, target: event.reason.line });
-  });
-
-  window.addEventListener('error', (event) => {
-    sampleRUM('error', { source: event.filename, target: event.lineno });
-  });
+  sampleRUM();
+  window.addEventListener('hlx:section:loaded', sampleRUM.enhance, { once: true });
```

Comment:
```suggestion
  window.addEventListener('aem:section:loaded', sampleRUM.enhance, { once: true });
```

Reply:
To be discussed, there is the idea that `hlx` namespace could be "internal" (i.e. do not rely on it, can change), `aem` being public (i.e. backward comp)...